cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# this script creates all the users/keys on the external cluster
+# those keys will be injected via the import-external-cluster.sh once this one is done running
+# so you can run import-external-cluster.sh right after this script
+set -Eeuo pipefail
+
+#############
+# VARIABLES #
+#############
+
+: "${CLIENT_CHECKER_NAME:=client.checker}"
+
+#############
+# FUNCTIONS #
+#############
+
+function is_available {
+  command -v "$@" &>/dev/null
+}
+
+function checkEnv() {
+  if ! is_available ceph; then
+    echo "'ceph' binary is expected'"
+    exit 1
+  fi
+
+  if ! ceph -s 1>/dev/null; then
+    echo "cannot connect to the ceph cluster"
+    exit 1
+fi
+}
+
+function createCheckerKey() {
+  checkerKey=$(ceph auth get-or-create "$CLIENT_CHECKER_NAME" mon 'allow r, allow command quorum_status'|awk '/key =/ { print $3}')
+  echo "export ROOK_EXTERNAL_CHECKER_SECRET=$checkerKey"
+  echo "export ROOK_EXTERNAL_CHECKER_USERNAME=$CLIENT_CHECKER_NAME"
```

Comment:
Instead of printing them to the console, how about writing them to a file? Then the keyrings won't be printed in plain text for all to see. Then importing the env variables can just be `source vars.sh` or similar.

Reply:
Hum, I think it's really a detail :-), one could argue that the keys are plain text in the file too so... If people don't want to see the output they can still redirect it to a file.
Also sourcing the file won't actually populate the env var in your current shell but in the one from the script for that, you would need to use `eval` but I just prefer to differ that to the end-user.

Reply:
Sure, it is a detail we can leave to the user.